plaso/cli/helpers/artifact_filters.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+"""The artifacts filter file CLI arguments helper."""
+
+from __future__ import unicode_literals
+
+import os
+
+from plaso.cli import tools
+from plaso.cli.helpers import interface
+from plaso.cli.helpers import manager
+from plaso.lib import errors
+
+
+class ArtifactFiltersArgumentsHelper(interface.ArgumentsHelper):
+  """Artifacts filter file CLI arguments helper."""
+
+  NAME = 'artifact_filters'
+  DESCRIPTION = 'Artifact filters command line arguments.'
+
+  @classmethod
+  def AddArguments(cls, argument_group):
+    """Adds command line arguments to an argument group.
+
+    This function takes an argument parser or an argument group object and adds
+    to it all the command line arguments this helper supports.
+
+    Args:
+      argument_group (argparse._ArgumentGroup|argparse.ArgumentParser):
+          argparse group.
+    """
+    argument_group.add_argument(
+        '--artifact_filters', '--artifact-filters',
```

Comment:
maybe split this in 2 different options, artifact_filters and artifact_filters_file?

Reply:
Done.